### ISSUE TITLE
[Errno 8] when running recipes

### ISSUE THREAD

**Issue by Chriskmpruitt** [autopkg/autopkg#243]
Hello!

I am new to AutoPKG, so i am sure i am just missing something easy.

But when I run Recipes via command line or the app. i get this error, 

Error running recipes

[Errno 8] nodename nor servname provided, or not known.

now when the app auto added the Casper distribution point, the wrong information was put in. So i have manually added our DP.

I have ran the following commands 

defaults read com.github.autopkg JSS_REPOS - Looks right
defaults read com.github.autopkg JSS_URL - Looks right
defaults read com.github.autopkg JSS_VERIFY_SSL - says SSL does not exist, but i know we have a trusted certificate 
defaults read com.github.autopkg API_USERNAME - looks right
defaults read com.github.autopkg API_PASSWORD - looks right

**Comment by arubdesu** [comment]
Hello, you're having an issue with the JSSImporter. Please use the docs there, and file an issue if you're still having difficulty https://github.com/sheagcraig/JSSImporter/issues

**Comment by macmule** [comment]
Just to say.. saw this earlier & commented at: https://github.com/lindegroup/autopkgr/issues/466 & found a solution via: sheagcraig/JSSImporter#91

For me, the distribution point had a space before it's name in this test JSS.

So it's worth verifying no unwanted spaces etc in the distribution point name etc in the JSS

**Comment by croaker-1** [comment]
We have been having this issue for just under a month and I finally tracked down the cause this afternoon. We user SMB shares for distribution points and are moving from standard Windows shares to DFS shares. I recently (just under a month ago) updated the URL on our jss to point to the DFS share. The difference between a standard share and a DFS share URL:

standard: smb://fileserver.domain.com/sharename
DFS: smb://domain.com/namespace/sharename

This means the the server name entry is 'domain.com/namespace' and I think the slash is confusing JSSImporter. Once I set the URL back to the standard format everything worked again.
